spec/driver_spec.rb

```diff
@@ -2674,8 +2676,11 @@ def which_for(character)
       end
 
       it "can allow specific hosts" do
-        driver.allow_url("example.com")
-        driver.allow_url("www.example.com")
+        configure do |config|
```

Comment:
To elaborate: there's a lot of hidden magic here due to the fact that the driver is lazily initialized via the `let` block (with the driver itself hidden within `AppRunner`.) Previously some of that magic was exposed via explicit configuration directives. Now, `AppRunner` transparently resets the configuration hash between specs via an include block and then configures the driver on first request due to RSpec magic. Again, the straightforwardness of the test is nice -- but it certainly confused me on first glance.